### ISSUE TITLE
le-import command broken, panics

### ISSUE THREAD

**Issue by schumann2k** [hlandau/acme#86]
I tried to import my old letsencrypt data with `acmetool import-le /etc/letsencrypt/` and it panic'ed:
```
/tmp/acmetool-v0.0.39-linux_386/bin# ./acmetool import-le /etc/letsencrypt/
panic: unsupported key type

goroutine 1 [running]:
github.com/hlandau/acme/storage.getPublicKey(0x8515020, 0x18a23128, 0x0, 0x0)
        /home/travis/gopath/src/github.com/hlandau/acme/storage/util.go:139 +0x106
github.com/hlandau/acme/storage.determineKeyIDFromKey(0x8515020, 0x18a23128, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/hlandau/acme/storage/util.go:144 +0x3d
github.com/hlandau/acme/storage.(*fdbStore).ImportKey(0x18aedf20, 0x8515020, 0x18a23128, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/hlandau/acme/storage/storage-fdb.go:724 +0x37
main.importKey(0xb7656b98, 0x18aedf20, 0x18c58a20, 0x2e, 0x0, 0x0)
        /home/travis/gopath/src/github.com/hlandau/acme/cmd/acmetool/le-import.go:94 +0xad
main.cmdImportLE()
        /home/travis/gopath/src/github.com/hlandau/acme/cmd/acmetool/le-import.go:43 +0x5ea
main.main()
        /home/travis/gopath/src/github.com/hlandau/acme/cmd/acmetool/main.go:146 +0x481

goroutine 5 [syscall]:
os/signal.loop()
        /home/travis/.gimme/versions/go1.5.3.linux.amd64/src/os/signal/signal_unix.go:22 +0x1a
created by os/signal.init.1
        /home/travis/.gimme/versions/go1.5.3.linux.amd64/src/os/signal/signal_unix.go:28 +0x36
```
I looked through git-blame and noticed that the last time that function was changed was December 23rd 2015 so I downloaded a version from before that date (0.0.23) and this time `import-le` finished without error.

**Comment by hlandau** [comment]
Whoops. Looks like some refactoring which changed an interface didn't get caught because crypto.PrivateKey is just interface{} and so will accept absolutely anything. I'll fix it ASAP. Should probably add tests for the import functionality.

**Comment by schumann2k** [comment]
Awesome, thanks. :) Great tool btw, feels much more solid than the official client imo. :D

**Comment by hlandau** [comment]
Fixed in v0.0.40.
